app/helpers/application_helper.rb

```diff
@@ -28,6 +28,29 @@ def current_path_without_query_string
     request.original_fullpath.split("?", 2).first
   end
 
+  def format_with_html_line_breaks(string)
+    ERB::Util.html_escape(string || "").strip.gsub(/(?:\r?\n)/, "<br/>").html_safe
+  end
+
+  def page_title(*title_parts)
+    # This helper may be called multiple times on the
+    # same page, with or without the necessary arguments
+    # to construct the title (e.g. on a nested form).
+    # rubocop:disable Rails/HelperInstanceVariable
+    if title_parts.any?
+      title_parts.push("Admin") if params[:controller].match?(/^admin\//)
+      title_parts.push("GOV.UK")
+      @page_title = title_parts.reject(&:blank?).join(" - ")
+    else
+      @page_title
+    end
+    # rubocop:enable Rails/HelperInstanceVariable
+  end
```

Comment:
I'm looking at commit f7434f239bf5b3e66a34286e65a4642e3cecbcaa - it seems everything above this line isn't actually used in this commit (or at least not explicitly). Could it be moved to its own commit where it can be read and understood in isolation?